Processing/Explore_New_Parks.Rmd

```diff
@@ -198,6 +199,7 @@ gw_gages <- gage_sites %>%
                        #values_from = mean_lev_va) #%>%
 
 ```  
+
```

Comment:
Yeah that's a Valles issue -- generally this would be run on multiple parks and if at least one has GW data, there is no error. Because it's an exploratory script, I'm not going to add an error check for that for now but we can in the future if we want.